src/NServiceBus.Core/Pipeline/Incoming/InvokeHandlerContext.cs

```diff
@@ -35,13 +36,13 @@ public InvokeHandlerContext(MessageHandler handler, string messageId, string rep
 
         public MessageMetadata MessageMetadata { get; }
 
-        public bool HandleCurrentMessageLaterWasCalled { get; private set; }
+        // remove in v8
```

Comment:
Can't we put obsoleteEx here as well in order to not forget to remove?

Reply:
Talked to @timbussmann about this, adding obsoleteex won't gain us anything since we won't get a compile error so no code us using this. I withdraw my comment